### DISCUSSION TITLE
Imperial Units in Android GUI

### DISCUSSION THREAD

**Discussion by troycarpenter** [owntracks/android#1571] [Q&A] [answered]
I have been looking for any post that may have asked this already and surprisingly came up empty.  Maybe I'm conflating some of the programs I've tried over the past few days, but here goes:

Is there any way to have the GUI present data in Imperial units?  Feet, miles, mph for example?  I don't mind the configuration units staying metric, but data presented to my users (distance, speed, altitude, etc) would be nice to be in something they understand.

Like I said, I may be misremembering, but I thought there was a setting for this.

Thanks

**Comment by growse** [comment, marked as answer]
This is a great idea. Would you be able to open this as an issue (rather than a discussion), that way I can stick it on the backlog.

Thanks!

**Reply by troycarpenter** [reply]
Done.  #1573

**Reply by jpmens** [reply]
I've added a cross-link to [iOS](https://github.com/owntracks/ios/issues/765)

**Comment by troycarpenter** [comment]
Thanks for the quick reply.  The whole Life360 debacle (mainly shutting out 3rd party apps via their API, namely Home Assistant) has pushed me to investigate OwnTracks.  I've got my whole family switched over, but we are still learning how to use it.  I have it integrated into Home Assistant, but I'm also looking into setting up the recorder and the web frontend.
